src/backend/feed.js

```diff
@@ -2,34 +2,31 @@ const storage = require('./utils/storage');
 
 class Feed {
   /**
-   * Saves current feed to appropriate set in Redis
+   * Saves current feed to Redis
    */
   async save() {
     this.feedId = await storage.addFeed(this);
+    await storage.storeFeed(this.feedId);
   }
 
   /**
    * Checks if feed is part active feeds list. If it is, will move feed from active FEEDS set to INACTIVE_FEEDS set
+   * else will move ffeed from INACTIVE_FEEDS to FEEDS
    */
-  async inactive() {
-    const isActive = await this.isActive(this);
+  async move() {
```

Comment:
So this is really `toggleStatus` vs. `move` right?

Reply:
I think you'd be better off having 2 functions for this vs. one.  Let the caller mange deciding which one to use.